astropy/units/format/generic.py

```diff
@@ -469,8 +476,8 @@ def _parse_unit(cls, s, detailed_exception=True):
     _translations = {
         '\N{GREEK SMALL LETTER MU}': '\N{MICRO SIGN}',
         '\N{MINUS SIGN}': '-',
-        '\N{OHM SIGN}': 'Ohm',
-        '\N{GREEK CAPITAL LETTER OMEGA}': 'Ohm',
+        '\N{GREEK CAPITAL LETTER OMEGA}': '\N{OHM SIGN}',
+        '\N{LATIN CAPITAL LETTER A WITH RING ABOVE}': '\N{ANGSTROM SIGN}'
```

Comment:
Not the other way around?

Reply:
I read that both these specialized versions are considered deprecated (at least the OHM SIGN) and should not be used anymore.

Reply:
See https://en.wiktionary.org/wiki/%CE%A9

Reply:
I thought inside `_parse_unit`, it was clearer if we were comparing with the more obvious 'OHM SIGN' and 'ANGSTROM SIGN'.